packages/protocol/scripts/foundry/start_anvil.sh

```diff
@@ -45,7 +45,7 @@ while ! nc -z localhost $ANVIL_PORT; do
 done
 
 # enabled logging
-cast rpc anvil_setLoggingEnabled true --rpc-url $ANVIL_RPC_URL
+cast rpc anvil_setLoggingEnabled $ANVIL_LOGGING_ENABLED --rpc-url $ANVIL_RPC_URL
```

Comment:
Can we keep the default as true?

Reply:
We can, but I freshly changed it to true based on @pahor167 request 😁